api/v1/router.py

```diff
@@ -0,0 +1,101 @@
+"""API routing and fallback logic."""
+import http
+import logging
+from urllib.parse import urlparse
+
+from fastapi.requests import Request
+from fastapi.responses import Response
+from fastapi.routing import APIRouter
+from httpx import AsyncClient
+
+from api.core.settings import settings
+
+
+log = logging.getLogger(__name__)
+router = APIRouter(prefix='/api')
+
+
+# Must be the last defined route.
```

Comment:
Nitpicking, I think this comment better fits in `main.py` where you include all the routers in the api.

Reply:
Why is that? The routes defined in `main` are not mounted to the `/api` prefix, in contrast to the routes here. Therefore, it doesn't really matter which routes are defined where, as long as they do not begin with `/api`. Or am I missing something?

Reply:
Ah right, I didn't notice they were mounted to `/api` prefix.